community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/KernelAPIWriteTestBase.java

```diff
@@ -43,12 +47,36 @@
 @SuppressWarnings( "WeakerAccess" )
 public abstract class KernelAPIWriteTestBase<WriteSupport extends KernelAPIWriteTestSupport>
 {
+    public class ConditionalTeardown implements TestRule
```

Comment:
This is an exact copy from the other test case. We really should extract it to its own utility (see other comment for the name of that utility).